app/views/showcase/_showcase_item.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if(left) %>
+<div class="showcase-item left-item">
```

Comment:
There are utility classes for left and right in Foundation. Do we have to write our own?